core/controllers/feedback_test.py

```diff
@@ -39,6 +42,28 @@
     'text', 'updated_status', 'updated_subject', 'received_via_email']
 
 
+class MockFeedbackAnalyticsAggregator(
+        feedback_jobs_continuous.FeedbackAnalyticsAggregator):
+    """A modified FeedbackAnalyticsAggregator that does not start a new batch
+    job when the previous one has finished.
+    """
+    @classmethod
+    def _get_batch_job_manager_class(cls):
+        return MockFeedbackAnalyticsMRJobManager
+
+    @classmethod
+    def _kickoff_batch_job_after_previous_one_ends(cls):
+        pass
+
+
+class MockFeedbackAnalyticsMRJobManager(
+        feedback_jobs_continuous.FeedbackAnalyticsMRJobManager):
+
+    @classmethod
+    def _get_continuous_computation_class(cls):
+        return MockFeedbackAnalyticsAggregator
```

Comment:
Now that this is resolved, let's go with the old strategy:

```python
class MockFeedbackAnalyticsAggregator(
        feedback_jobs_continuous.FeedbackAnalyticsAggregator):
    """A modified FeedbackAnalyticsAggregator that does not start a new batch
    job when the previous one has finished.
    @classmethod
    def _kickoff_batch_job_after_previous_one_ends(cls):
        pass

# ...

with self.swap(feedback_jobs_continuous, 'FeedbackAnalyticsAggregator', MockFeedbackAnalyticsAggregator):
  # use continuous job as usual.

Reply:
Done